### ISSUE TITLE
ACID(21) zoom: warning: division by zero

### ISSUE THREAD

**Issue by PeterPablo** [matlab2tikz/matlab2tikz#664]
Executing `ACID(21)` in octave yields many `warning: division by zero` and might crash octave (and thus `testGraphical()`).
Tested on 37d139129f59ba73273ad3826b0a7e59856326db.

**Comment by PeterPablo** [comment]
```
diff --git a/test/suites/ACID.m b/test/suites/ACID.m
index 3118efd..b9db9fd 100644
--- a/test/suites/ACID.m
+++ b/test/suites/ACID.m
@@ -582,7 +582,6 @@ function [stat] = zoom()
                         'and \texttt{movePointsCloser()} ', ...
                         'of \texttt{cleanfigure()}.'];
     stat.unreliable = isOctave; %FIXME: investigate
-    %FIXME: this generates many "division by zero" in Octave
     stat.issues = [226,392,400];
 
     % Setup
@@ -601,7 +600,7 @@ function [stat] = zoom()
     plot(1:10,10:-1:1,'-r*',1:10,repmat(9,1,10),'-g*',[5.5,5.5],[1,9],'-b*')
     stairs(1:10,'-m*')
     xlim([2, 8.5]), ylim([2,7.5])
-    cleanfigure()
+    %cleanfigure()  %FIXME: this generates many "division by zero" in Octave
     plot([2,8.5,8.5,2,2],[2,2,7.5,7.5,2],'--k')
     xlim([0, 15]), ylim([0,10])
 
@@ -611,7 +610,7 @@ function [stat] = zoom()
     hold on
     plot(1:10,10:-1:1,'-r*',1:10,repmat(9,1,10),'-g*',[5.5,5.5],[1,9],'-b*')
     stairs(1:10,'-m*')
-    xlim([2, 8.5]), ylim([2,7.5])
+    %xlim([2, 8.5]), ylim([2,7.5])  %FIXME: this generates many "division by zero" in Octave
 end
 % =========================================================================
 function [stat] = bars()
```
... makes the warnings vanish, though would disable the `cleanfigure()` call and setting the axis limits.

Any ideas on how to solve this? Adding `drawnow` at various places does not help.

**Comment by PeterPablo** [comment]
A fix for the "division by zero" is presented in #665, though the testsuite execution might still hang octave due to some other issue (possibly due to the renderer).

**Comment by egeerardyn** [comment]
Closed by #665.
